### PR TITLE
Lovelace 0.101

### PR DESCRIPTION
**Pull request by iantrich**
**Description:**

https://github.com/home-assistant/home-assistant-polymer/pull/4117

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/lovelace/changelog.markdown [hunk 1]
```diff
@@ -2,23 +2,70 @@
...
+- 📣 [alarm panel card]: New `theme` option [#4057](https://github.com/home-assistant/home-assistant-polymer/pull/4057) @iantrich
+- 📣 [picture card]: New `theme` option [#4057](https://github.com/home-assistant/home-assistant-polymer/pull/4057) @iantrich
+- 📣 [picture elements card]: New `theme` option [#4057](https://github.com/home-assistant/home-assistant-polymer/pull/4057) @iantrich
+- 📣 [picture entity card]: New `theme` option [#4057](https://github.com/home-assistant/home-assistant-polymer/pull/4057) @iantrich
+- 📣 [picture glance card]: New `theme` option [#4057](https://github.com/home-assistant/home-assistant-polymer/pull/4057) @iantrich
+- 📣 [vertical stack card]: Add `title` option [#3839](https://github.com/home-assistant/home-assistant-polymer/pull/3839) @iantrich
+- 📣 [horizontal stack card]: Add `title` option [#3839](https://github.com/home-assistant/home-assistant-polymer/pull/3839) @iantrich
+- ⚠️ revert lovelace selectable text [#4095](https://github.com/home-assistant/home-assistant-polymer/pull/4095) @iantrich
+- 🔧 Fix delete button styling in hui-edit-view [#4102](https://github.com/home-assistant/home-assistant-polymer/pull/4102) @bramkragten
```
Comment (bramkragten) [inline source/lovelace/changelog.markdown]: This can be left out, is not changed in this release

File: source/lovelace/changelog.markdown [hunk 2]
```diff
@@ -2,23 +2,70 @@
...
+- 🔧 Added title to view configuation [#4037](https://github.com/home-assistant/home-assistant-polymer/pull/4037) @drakeloud
+- 🔧 Added view title to heading of Lovelace Add Card picker [#4083](https://github.com/home-assistant/home-assistant-polymer/pull/4083) @springstan
+- 🔧 Fix issue where long states do not wrap [#4101](https://github.com/home-assistant/home-assistant-polymer/pull/4101) @pdong
+- 🔧 Changed View Configuration icons to text [#4067](https://github.com/home-assistant/home-assistant-polymer/pull/4067) @drakeloud
+- 🔧 Change ha-device-picker to combo box + improve name handling + show area [#4089](https://github.com/home-assistant/home-assistant-polymer/pull/4089) @bramkragten
```
Comment (bramkragten) [inline source/lovelace/changelog.markdown]: This is not Lovelace

File: source/lovelace/changelog.markdown [hunk 3]
```diff
@@ -2,23 +2,70 @@
...
+- 🔧 Changed outdated "GUI editor" reference to "visual editor" [#4068](https://github.com/home-assistant/home-assistant-polymer/pull/4068) @SeanPM5
+- 🔧 Hide "Save" on card configuration modal if no card is picked [#4062](https://github.com/home-assistant/home-assistant-polymer/pull/4062) @pdong
+- 🔧 [light card]: Add width property [#3972](https://github.com/home-assistant/home-assistant-polymer/pull/3972) @shbatm
+- 🔧 Made thermostat history graph localizable [#4004](https://github.com/home-assistant/home-assistant-polymer/pull/4004) @springstan
+- 🔧 [thermostat card]: Convert thermostat to round-slider [#3734](https://github.com/home-assistant/home-assistant-polymer/pull/3734) @iantrich
+- 🔧 Update alarm more-info dialog button styles to match lovelace card [#3955](https://github.com/home-assistant/home-assistant-polymer/pull/3955) @joshmcrty
+- 🔧 [alarm panel card]: number buttons outline and adjusted padding and font-size [#3991](https://github.com/home-assistant/home-assistant-polymer/pull/3991) @mr-awk
+- 🔧 Translate Advanced mode toggle [#3997](https://github.com/home-assistant/home-assistant-polymer/pull/3997) @springstan
```
Comment (bramkragten) [inline source/lovelace/changelog.markdown]: Not Lovelace

**Review by bramkragten — COMMENTED** [review]
Should we only mention the changes that are relevant for the config of the users and the cards? Like new config options etc. instead of mentioning that we changed the headers of the editors?

**Comment by iantrich** [comment]
@bramkragten 🤷‍♂ probably too much detail, yeah. I'll trim it down

**Comment by iantrich** [comment]
@balloob or @frenck, can we get this merged so beta testers know what changes they should be looking at? Thanks.

**Pull request merged**